randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -554,6 +554,8 @@ Extra - asset_id: collision_camera_007
   * Extra - right_shield_def: None
   > Door to Navigation Station
       After Ghavoran - Super Missile Rotatable
+  > Dock from Spider Magnet Elevator
+      Before Ghavoran - Super Missile Rotatable
```

Comment:
Remove this. You can only access it after the flipper has been used. There's also no point in even having it as you can't go through it anyway